man/rgbasm.5

```diff
@@ -1398,6 +1398,16 @@ In ROM sections, it will be filled with the value passed to the
 .Fl p
 command-line option, except when using overlays with
 .Fl O .
+.Pp
+Instead of a specific number of bytes, you can specify
+.Ic DS ALIGN Ns Bq Ar align , offset
+to allocate however many bytes are required to align the subsequent data; see
```

Comment:
The way I see it, it's not clear what happens with the padding bytes — I assume that they're filled with the padding value, but are they? Is `ds align[3], $42` valid? What if the data is already aligned and you don't need any padding at all?
There are obvious answers to these questions, but I'd rather they were spelled out...

Reply:
Yeah, I think it'd be best to word this to make it clear it's only a different way to specify the length and only the length. I'll try to think of something tomorrow.

Reply:
> Instead of an exact number of bytes, you can specify `ALIGN[align, offset]` to allocate however many bytes are required to align the subsequent data. Thus, `DS ALIGN[align, offset], ...` is equivalent to `DS N, ...` followed by `ALIGN align, offset`, where `N` is the minimum value needed to satisfy the `ALIGN` constraint (see "Requesting alignment" below).

Reply:
We already specify above that empty `DS` space:
> In ROM sections, it will be filled with the value passed to the `-p` command-line option, except when using overlays with `-O`.
Maybe that needs clarifying in this section, regardless of whether you do `DS n` or `DS ALIGN[n]`.

Reply:
Does the above rewrite look OK?

Reply:
Okay @ISSOtm , that's rewritten:
![image](https://github.com/gbdev/rgbds/assets/35663410/e0d51ad0-f50d-4334-bf6f-0cbe9042d1c2)

Reply:
I think the content is good!

However, the code snippets don't stand out enough IMO (and it'll be worse in a non-monospace font). I'd suggest enclosing them in at least `Sq` (alternatively, `So`+`Sc`) or `Ql` (which loses any special formatting), and considering `D1` or `Dl` respectively.

<details><summary>Proposed alternatives</summary>

```man
Thus,
.Sq Ic DS ALIGN Ns Bo Ar align , offset Bc , No ...
is equivalent to
.Sq DS Ar n , No ...
followed by
.Sq ALIGN Ns Bq Ar align , offset ,
where
```

```man
Thus,
.Ql DS ALIGN[align, offset], ...
is equivalent to
.Ql DS n, ...
followed by
.Ql ALIGN align, offset
where
```

```man
Thus,
.D1 Ic DS ALIGN Ns Bo Ar align , offset Bc , No ...
is equivalent to
.D1 DS Ar n , No ...
followed by
.D1 ALIGN Ns Bq Ar align , offset ,
where
```

```man
Thus,
.Dl DS ALIGN[align, offset], ...
is equivalent to
.Bd -literal -offset Ds
DS n, ...
ALIGN align, offset
.Ed
where
```

</details>

Reply:
I like that first `Sq` option. Committed.